blockware/bouncing-logo/platformio.ini

```diff
@@ -28,3 +28,11 @@ upload_speed = 460800
 
 build_flags = !echo "-DESP8266=1 -DLOGGING=1 -DGIT_REVISION='\"$(git rev-parse HEAD)\"'"
 monitor_speed = 115200
+
+[env:VirtualBlox]
```

Comment:
When you want to support VirtualBlox for a blockware, you'll just need to add this to the platformio.ini